server/views/pages/barcode/choose-barcode-option.njk

```diff
@@ -19,35 +19,37 @@
       }) }}
     {% endif %}
 
-    <form action="/barcode/choose-barcode-option" method="post" novalidate id="choose-barcode-option-form">
-      <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-
-      {% call govukFieldset({ }) %}
-
-        <div class="govuk-grid-row">
-          <div class="govuk-grid-column-two-thirds">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-              <h1 class="govuk-fieldset__heading">How do you want to print the barcodes?</h1>
-            </legend>
-            <p class="govuk-hint">Each recipient's address will be displayed with the barcode. You'll need to use a window envelope.</p>
-          </div>
-        </div>
-
-        {{ govukRadios({
-          idPrefix: "barcodeOption",
-          name: "barcodeOption",
-          items: ['image', 'coversheet'] | renderChooseBarcodeOptionRadiosFilter,
-          errorMessage: errors | findError('barcodeOption')
-        }) }}
-
-        {{ govukButton({
-          text: "Continue",
-          preventDoubleClick: true,
-          attributes: { 'data-qa': 'continue-button' }
-        }) }}
-
-      {% endcall %}
-    </form>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <form action="/barcode/choose-barcode-option" method="post" novalidate id="choose-barcode-option-form">
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+            {{ govukRadios({
+              fieldset: {
+                legend: {
+                  text: 'How do you want to print the barcodes?',
+                  isPageHeading: true,
+                  classes: 'govuk-fieldset__legend--xl'
```

Comment:
For pages with a single question use the govuk components to set page headings as these are built with accessibility in mind - we created our own headings and created visually hidden labels that didn't match visual labels thus breaking some screen reader functionality linking labels to inputs.